integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/SearchPredicateIT.java

```diff
@@ -37,8 +37,8 @@
 
 public class SearchPredicateIT {
 
-	private static final String INDEX_NAME = "IndexName";
-	private static final String ANOTHER_INDEX_NAME = "AnotherIndexName";
```

Comment:
See. You had to do this because you used the Elasticsearch index names, not the Hibernate Search index names.

This change shouldn't be necessary. Please remove this change and use the correct index names in the Elasticsearch backend.